component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/smsotp/SMSOTPUtils.java

```diff
@@ -105,9 +105,10 @@ public static boolean isSMSOTPDisableForLocalUser(String username, Authenticatio
      * @param username  the Username
      * @param attribute the Attribute
      * @throws SMSOTPException
+     * @throws UserStoreException
      */
     public static void updateUserAttribute(String username, Map<String, String> attribute, String tenantDomain)
-            throws SMSOTPException {
+            throws SMSOTPException, UserStoreException {
```

Comment:
remove API changes

Reply:
Can this util class be used by other mobiles other than sms otp authenticator. Because all the places this method reffering has changed